.github/workflows/CI-e2e-notebooks.yml

```diff
@@ -0,0 +1,124 @@
+name: CI e2e notebooks
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  ui-build:
+    env:
+      node-version: 16.8
+
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js ${{ env.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.node-version }}
+
+      - name: Install yarn
+        run: npm install yarn -g
+
+      - name: Install yarn dependencies
+        run: yarn install --frozen-lock-file
+
+      - name: Build Typescript
+        run: yarn buildall
+
+      - name: Upload the build result
+        uses: actions/upload-artifact@v3
+        with:
+          name: raiwidgets-js
+          path: raiwidgets/raiwidgets/widget
+
+  ci-e2e-notebook:
+    needs: ui-build
+
+    env:
+      node-version: 16.8
+
+    strategy:
+      matrix:
+        operatingSystem: [ubuntu-latest, windows-latest]
```

Comment:
Seems like these tests aren't being run on MAC OS> Could we add a comment that these need to be added at some point. I know @vinuthakaranth has a work item for re-enabling these tests for MAC OS>

Reply:
Certainly! Just added a TODO comment right above that line.